### DISCUSSION TITLE
Example conflict solver ignore rules.

### DISCUSSION THREAD

**Discussion by LordOfLA** [bcssov/IronyModManager#156] [Show and tell]
These are the current rules I use to ignore certain conflicts in the conflict solver. Can cut conflict count you need to care about by quite a bit. Naturally one should tweak as needed for one's goals.

flags
common\\random_names
!common\\random_names\base
common\\scripted_loc
gfx\\event_pictures
gfx\\dx11_fixes
gfx\\interface
gfx\\loadingscreens
gfx\\map
gfx\\models
!gfx\\models\\planets
gfx\\portraits
!gfx\\portraits\\asset_selectors
gfx\\projectiles
gfx\\ui_overhaul_qhd
localisation\\braz_por
localisation\\french
localisation\\german
localisation\\polish
localisation\\russian
localisation\\simp_chinese
localisation\\spanish
localisation\\\*braz_por*.yml
localisation\\\*chinese*.yml
localisation\\\*french*.yml
localisation\\\*german*.yml
localisation\\\*polish*.yml
localisation\\\*russian*.yml
localisation\\\*spanish*.yml

**Comment by bcssov** [comment]
Post your ignore rules for others to use here.

**Reply by Arkhorse** [reply]
[Stellaris - Ignore Rules Irony Mod Manager.txt](https://github.com/bcssov/IronyModManager/files/6216385/Stellaris.-.Ignore.Rules.Irony.Mod.Manager.txt)

I use this one alot. Ive found patching most of those mods can cause unexpected issues, so I just ignore them for now :P
